control/init.sh

```diff
@@ -11,15 +11,19 @@ fi
 
 
 function realpath() {
-    python3 -c "import os,sys; print(os.path.realpath(sys.argv[1]))" $1
+    python -c "import os,sys; print(os.path.realpath(sys.argv[1]))" $1
```

Comment:
Was this change necessary? If a virtualenv is activated it will work, but if a virtualenv is not activated I think this will fail on any modern Ubuntu.

Reply:
I mainly am aiming for consistency, whether that is using `python` or `python3`. Maybe I'm misunderstanding how this would be used, but I assumed it would only be used when a virtualenv is activated, otherwise the operator does not have a valid setup. We use `python` [here](https://github.com/dimagi/commcare-cloud/blob/9826971801730c3b603791247260dfebb5a7ab7f/control/init.sh#L98) and [here](https://github.com/dimagi/commcare-cloud/blob/9826971801730c3b603791247260dfebb5a7ab7f/control/init.sh#L127). Maybe those are different cases than this though?

Reply:
Sure, if the virtualenv is always expected to be activated when this is called then it's fine. It's also fine from a consistency perspective. This utility may have been written and possibly copied from a place where it was expected to work outside of a virtualenv context.

Incidentally, when this was [originally added](https://github.com/dimagi/commcare-cloud/commit/50cceea14a54ba4639907adde80e39884117e69b) it was called before the virtualenv was activated. However when it was later [changed from `python` to `python3`](https://github.com/dimagi/commcare-cloud/commit/a5d69d593722e5bd55148aa60d2b33e0466c68cf) it was also called after the virtualenv was activated, so that change was not strictly necessary (and that commit was inconsistent in changing `python` to `python3`).